src/activities/components/ActivityItem.vue

```diff
@@ -134,9 +134,30 @@
         flat
         no-caps
         align="between"
+        type="a"
         color="secondary"
         :class="{ 'full-width': $q.platform.is.mobile }"
-        class="open-conversation-button"
+        class="activity-item-button"
+        :href="`/api/activities-ics/${activity.id}/`"
+      >
+        <template #default>
+          <div>
+            <QIcon
+              name="fas fa-calendar-plus"
+              size="xs"
+              class="q-mr-xs icon-left"
+            />
+            <span>{{ $t('ACTIVITYLIST.ITEM.EXPORT_ICS') }}</span>
+          </div>
+        </template>
+      </QBtn>
+      <QBtn
+        flat
+        no-caps
+        align="between"
+        color="secondary"
+        :class="{ 'full-width': $q.platform.is.mobile }"
```

Comment:
On mobile with two buttons, they are stacked, but it seems there would be enough room for two side-by-side?

![photo_2021-04-02_12-32-58 small](https://user-images.githubusercontent.com/31616/113412264-bc247d80-93af-11eb-91ac-d227c9323d7d.jpg)

Reply:
very true. if you decide this is the right place for the button (comment below) then i fix this

Reply:
Buttons are side-by-side now. And both are always to the right side. See Screenshots in comment below.